app/javascript/components/widgets/widgets/forest-change/tree-loss-plantations/actions.js

```diff
@@ -10,11 +10,12 @@ export default ({ params }) =>
     .then(
       axios.spread((plantationsloss, gadmLoss) => {
         let data = {};
-        const loss = plantationsloss.data && plantationsloss.data.data;
+        const lossPlantations =
+          plantationsloss.data && plantationsloss.data.data;
         const totalLoss = gadmLoss.data && gadmLoss.data.data;
-        if (loss.length && totalLoss.length) {
+        if (lossPlantations.length && totalLoss.length) {
```

Comment:
if `lossPlantations` is undefined then this will crash also. You need to check for it first before doing an array method.

Reply:
`if (lossPlantations && totalLoss && lossPlantations.length && totalLoss.length) {` ?